scripts/ci/docker-compose/integration-otel.yml

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+version: "3.7"
+services:
+  otel-collector:
+    image: otel/opentelemetry-collector:0.70.0
+    container_name: "breeze-opentelemetry-collector"
+    ports:
+      - "1888:1888"     # pprof extension
+      - "28888:8888"    # Prometheus metrics exposed by the collector
+      - "28889:8889"    # Prometheus exporter metrics
+      - "13133:13133"   # health_check extension
+      - "4317:4317"     # OTLP gRPC receiver
+      - "4318:4318"     # OTLP http receiver
+      - "55679:55679"   # zpages extension
+
+  prometheus:
+    image: prom/prometheus
+    container_name: "breeze-prometheus"
+    user: "0"
+    ports:
+      - "29090:9090"
+    volumes:
+      - ./prometheus-config.yml:/etc/prometheus/prometheus.yml
+      - ./prometheus/volume:/prometheus
+
+  grafana:
+    image: grafana/grafana:8.2.4
+    container_name: "breeze-grafana"
+    environment:
+      GF_AUTH_ANONYMOUS_ENABLED: true
+      GF_AUTH_ANONYMOUS_ORG_NAME: "Main Org."
+      GF_AUTH_ANONYMOUS_ORG_ROLE: "Admin"
+      GF_PATHS_PROVISIONING: /grafana/provisioning
+    ports:
+      - "23000:3000"
+    volumes:
+      - ./grafana/volume/data:/grafana
+      - ./grafana/volume/datasources:/grafana/datasources
+      - ./grafana/volume/dashboards:/grafana/dashboards
+      - ./grafana/volume/provisioning:/grafana/provisioning
+
+  airflow:
+    environment:
+      - INTEGRATION_STATSD=true
+      - AIRFLOW__METRICS__STATSD_ON=True
+      - AIRFLOW__METRICS__STATSD_HOST=statsd-exporter
+      - AIRFLOW__METRICS__STATSD_PORT=9125
```

Comment:
Are these statsd values being set temporary? Since this is the OTEL integration I would expect to see OTEL env vars here not statsd

Reply:
Correct, at the moment there is no code in Airflow that would accept other values so I left these for now, but in hindsight that doesn't make any sense I guess.  I'll remove them  and leave a note to see the statsd one for formatting once they are implemented